src/main/java/spoon/SpoonAPI.java

```diff
@@ -22,7 +24,12 @@
 	 * Adds a processor (fully qualified name).
 	 */
 	public void addProcessor(String name);
-	
+
+	/**
+	 * Adds a instance of a processor.
```

Comment:
"an instance' the user is responsible for keeping a pointer to it for later retrieving some processing information